src/GeneralStateTestsFiller/VMTests/vmTests/push0GasFiller.yml

```diff
@@ -0,0 +1,56 @@
+# EIP-3855: PUSH0 instruction - test for gas price
+push0Gas:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 10000000
+      code: ''
+      nonce: '0'
+      storage: {}
+
+    0000000000000000000000000000000000000100:
+      balance: 0
+      code: ':raw 0x5f'
```

Comment:
save gas before,  do (PUSH0) in lllc  https://github.com/winsvega/solidity (add as a PR here, retesteth docker uses it)
save the gas after
sstore the gas used change into storage.